integrations/gradle-bloop/src/main/scala/bloop/integrations/gradle/model/BloopConverter.scala

```diff
@@ -71,8 +72,11 @@ final class BloopConverter(parameters: BloopParameters) {
     val classpath: List[Path] = {
       // Cannot use `sourceSet.getRuntimeClasspath` because returns jars of subprojects, need dirs!
       val projectDependencyClassesDirs =
-        projectDependencies.map(dep => getClassesDir(dep.getDependencyProject, sourceSet))
-      (projectDependencyClassesDirs ++ dependencyClasspath.map(_.getFile.toPath)).toList
+        projectDependencies.map { dep =>
+          val project = dep.getDependencyProject
+          getClassesDir(dep.getDependencyProject, project.getSourceSet(parameters.mainSourceSet))
```

Comment:
`getClassesDir(project, project.getSourceSet(parameters.mainSourceSet))`?

Reply:
thanks -- fixed in https://github.com/scalacenter/bloop/pull/614/commits/5fe9e9ce8cb7b28a5d281e9c547b2c230f274ce2